docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Perhaps, use shape if you would like to be "explicit" instead of "strict"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>